x509util/x509util.go

```diff
@@ -861,6 +861,34 @@ func ExtractSCT(sctData *x509.SerializedSCT) (*ct.SignedCertificateTimestamp, er
 	return &sct, nil
 }
 
+// ParseSCTListIntoSCTs serializes SCTs into SCT list.
+func ParseSCTListIntoSCTs(scts []*ct.SignedCertificateTimestamp) (*x509.SignedCertificateTimestampList, error) {
+	var sctList x509.SignedCertificateTimestampList
+	sctList.SCTList = []x509.SerializedSCT{}
+	for i, sct := range scts {
+		sctData, err := EncodeSCT(sct)
+		if err != nil {
+			return nil, fmt.Errorf("error serializing SCT number %d: %s", i, err)
+		}
+		sctList.SCTList = append(sctList.SCTList, *sctData)
+	}
+	return &sctList, nil
+}
+
+// EncodeSCT serializes an SCT into a TLS-encoded SCT.
+func EncodeSCT(sct *ct.SignedCertificateTimestamp) (*x509.SerializedSCT, error) {
```

Comment:
This might not need to be exported. It might be clearer to remove it and inline it in the caller unless it has other uses. 

If it's staying it could return just the result of Marshal and create the struct where it can be fully initialized above.

Reply:
Done